src/apps/stable/routes/session/forgotPassword/index.tsx

```diff
@@ -0,0 +1,128 @@
+import Input from 'elements/emby-input/Input';
+import { useMutation } from '@tanstack/react-query';
+import Dashboard from 'utils/dashboard';
+import { getUserApi } from '@jellyfin/sdk/lib/utils/api/user-api';
+import { ForgotPasswordAction } from '@jellyfin/sdk/lib/generated-client/models/forgot-password-action';
+import ServerConnections from 'components/ServerConnections';
+
+export const ForgotPasswordPage = () => {
+    const navigate = useNavigate();
+    const [username, setUsername] = useState('');
+
+    const forgotPasswordMutation = useMutation({
+        mutationFn: async (enteredUsername: string) => {
+            const currentApi = ServerConnections.getCurrentApi();
+            if (!currentApi) {
+                throw new Error('API not available');
+            }
+            const response = await getUserApi(currentApi).forgotPassword({
+                forgotPasswordDto: {
+                    EnteredUsername: enteredUsername
+                }
+            });
+
+            return response.data;
+        },
+        onSuccess: (result) => {
+            if (result.Action == ForgotPasswordAction.ContactAdmin) {
+                Dashboard.alert({
+                    message: globalize.translate('MessageContactAdminToResetPassword'),
+                    title: globalize.translate('ButtonForgotPassword')
+                });
+            }
+
+            if (result.Action == ForgotPasswordAction.InNetworkRequired) {
+                Dashboard.alert({
+                    message: globalize.translate('MessageForgotPasswordInNetworkRequired'),
+                    title: globalize.translate('ButtonForgotPassword')
+                });
+            }
+
+            if (result.Action == ForgotPasswordAction.PinCode) {
+                let msg = globalize.translate('MessageForgotPasswordFileCreated');
+                msg += '<br/><br/>';
+                msg += globalize.translate('MessageForgotPasswordPinReset');
+                msg += '<br/><br/>';
+                msg += result.PinFile;
+                msg += '<br/>';
+                Dashboard.alert({
+                    message: msg,
+                    title: globalize.translate('ButtonForgotPassword'),
+                    callback: function () {
+                        navigate('/forgotpasswordpin');
+                    }
+                });
+            }
+        }
+    });
+
+    const handleCancel = useCallback(() => {
+        navigate(-1);
+    }, [navigate]);
+
+    const handleSubmit = useCallback(async (e: React.FormEvent) => {
+        e.preventDefault();
+        forgotPasswordMutation.mutate(username);
+    }, [username, forgotPasswordMutation]);
+
+    const handleUsernameChange = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+        setUsername(e.target.value);
+    }, []);
+
+    return (
+        <Page
+            id='forgotPasswordPage'
+            className='page standalonePage forgotPasswordPage mainAnimatedPage'
```

Comment:
The `Page` componenet automatically adds the `page` class.

```suggestion
            className='standalonePage forgotPasswordPage mainAnimatedPage'
```